lib/flutter_sidebar.dart

```diff
@@ -26,10 +26,11 @@ class Sidebar extends StatefulWidget {
   }) : super(key: key);
 
   @override
-  _SidebarState createState() => _SidebarState();
+  _SidebarState createState() => _SidebarState(activeTabIndices);
 }
 
 class _SidebarState extends State<Sidebar> with SingleTickerProviderStateMixin {
+  _SidebarState(this.activeTabIndices);
```

Comment:
Why pass it as a parameter when the value is already available under `widget.activeTabIndices`?

I'd have fixed it by doing 
```dart
activeTabIndices = widget.activeTabIndices;
```
at the top of `initState` instead. Any particular reason for this?

Reply:
I'm sorry. I was not familiar with Stateful Widget.

I totally agree with what you are saying.So I'll push the commit I modified later.